tests/Bridge/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactoryTest.php

```diff
@@ -335,4 +337,35 @@ public function testCreateWithPropertyFormatRestriction(): void
             $this->assertEquals($format, $schema['format']);
         }
     }
+
+    public function testCreateWithCompositeConstraint(): void
```

Comment:
```suggestion
    public function testCreateWithSequentiallyConstraint(): void
```
To be clearer :slightly_smiling_face:

Reply:
Thanks, updated :)